src/openms/source/FORMAT/MzTabFile.cpp

```diff
@@ -644,7 +644,7 @@ namespace OpenMS
         s.substitute("ms_run[","").substitute("]","");
         vector<String> ms_run;
         s.split(',', ms_run);
-        for (auto a : ms_run)
+        for (auto& a : ms_run)
         {
           a.trim();
           mz_tab_metadata.assay[n].ms_run_ref.push_back(a.toInt());
```

Comment:
@timosachsenberg this seems reasonable since `ms_run` is a temporary variable, so modifying it should not have any side effects.